src/PickerDropdown/index.js

```diff
@@ -33,7 +33,9 @@ function PickerDropdown({ isOpen, suggestions, maxHeight, anchorElement, ...othe
         const dropdownStyle = { ...DROPDOWN_STYLE, ...dropdownPositionStyle, ...dropdownMaxHeightStyle };
         return (
             <Paper square style={ dropdownStyle }>
-                <PickerSuggestions suggestions={ suggestions } { ...otherProps } />
+                <MenuList>
```

Comment:
Rather than add an extra layer of nesting (and pull in another component from the mui library), I think it would be better to just apply the `menu` role to the existing parent `Paper`.

Reply:
The problem with doing that is that it then misses one of the core reasons for this change... Yes we get the role, but we'd lose the ```<ul>``` :( . So hopefully you won't mind if I persist pulling in and using ```<MenuList>```

Reply:
I don't understand why the `<ul>` is necessary - screenreaders etc respect aria roles, so this should be enough as long as you choose the appropriate role. Is this rule coming from a particular tool or linter? If so, maybe I should add it to this project!

As a compromise, I suggest adding `component="ul"` to the `<Paper>` element - that should do the trick without making the DOM any more complex. See https://material-ui.com/api/paper/ - most MUI elements can be switched to use a different HTML element type.

Reply:
Adding component=ul seems to be fine. Our undertstanding is that without the parent ```<ul>``` then screenreaders, and also it seems certain test automation tools struggle to understand which list a given "unenclosed" ```<li>``` belongs to. See https://stackoverflow.com/questions/2031725/is-using-li-without-enclosing-ul-tags-dangerous

I'm not sure what else I can add - other than the previous typeahead component we were using had the ```<li>``` elements wrapped in a ```<ul>``` - and we had no such problems ;-)

Reply:
Looks good - I'll merge and publish this today as 1.8.3.